commerce_coordinator/apps/commercetools/views.py

```diff
@@ -82,3 +90,51 @@ def post(self, request):
         send_order_confirmation_email(lms_user_id, customer.email, canvas_entry_properties)
 
         return Response(status=status.HTTP_200_OK)
+
+
+class OrderSanctionedView(APIView):
+    """View to sanction an order and deactivate the lms user"""
+    permission_classes = [IsAdminUser]
+
+    # authentication_classes = [] TODO: Update this with OAuth authentication
+
+    def post(self, request):
+        """
+        Receive a message from commerce tools forwarded by aws event bridge
+        to sanction order and deactivate user through LMS
+        """
+        input_data = {
+            **request.data
+        }
+
+        logger.debug('[CT-OrderSanctionedView] Message received from commercetools with details: %s', input_data)
+
+        message_details = OrderMessageInputSerializer(data=input_data)
+        message_details.is_valid(raise_exception=True)
+
+        client = CommercetoolsAPIClient()
+        order_id = message_details.data['order_id']
+        order = client.get_order_by_id(order_id)
+        order_workflow_state = order.state
+
+        customer = client.get_customer_by_id(order.customer_id)
```

Comment:
Do we need to catch any API response that raises an error and add an exception log?